README.md

```diff
@@ -213,6 +213,33 @@ Despite express-graphql being a reference implementation, GraphQL Server is actu
 
 That said, GraphQL Server is heavily inspired by express-graphql (it's the reference implementation after all). Rather than seeing the two as competing alternatives, we think that they both have separate roles in the GraphQL ecosystem: express-graphql is a reference implementation, and GraphQL Server is a GraphQL server to be used in production and evolve quickly with the needs of the community. Over time, express-graphql can adopt those features of GraphQL Server that have proven their worth and become established more widely.
 
+### application/graphql requests
+
+express-graphql supports the `application/graphql` Content-Type for requests, which is an alternative to `application/json` request with only the query part sent as text. Because GraphQL server does not parse request bodies, you need to handle that in a separate middleware. Here's an example for express:
```

Comment:
Might be helpful to say something about, "in the same way that we use bodyparser.json to parse application/json requests for graphql-server, we can use bodyParser.text plus one extra step in order to also parse application/graphql requests." Ie, to clarify that using a bodyparser middleware is not a new thing unique to application/graphql -- in normal graphql-server usage, we're already using a bodyparser middleware to parse application/json.